.github/workflows/require-core-maintainer-approval.js

```diff
@@ -19,10 +19,24 @@ module.exports = async ({ github, context, core }) => {
     ({ state, user: { login } }) => state === "APPROVED" && CORE_MAINTAINERS.has(login)
   );
   if (!maintainerApproved) {
-    const maintainerList = Array.from(CORE_MAINTAINERS)
-      .map((maintainer) => `${maintainer}`)
+    const marker = "<!-- MAINTAINER_APPROVAL -->";
+    const comments = await github.paginate(github.rest.issues.listComments, {
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      issue_number: context.issue.number,
+    });
+    const maintainers = Array.from(CORE_MAINTAINERS)
+      .map((maintainer) => `\`${maintainer}\``)
       .join(", ");
-    const message = `This PR requires an approval from at least one of core maintainers: ${maintainerList}.`;
+    const message = `This PR requires approval from at least one core maintainer (${maintainers}). If you're not sure who to request a review from, assign \`mlflow-automation\`.`;
```

Comment:
Q: Shall we skip showing this comment if a maintainer is already in the current reviewers list (not approved yet)?

Reply:
sounds good! let's skip it